### PR TITLE
Handle the annotatedBy field of policy metadata

### PR DESCRIPTION
**Pull request by flavio**
Ensure kwctl writes its version when annotating a policy.

At the same time, ensure `inspect`ion of a policy shows who annotated it.

### CODE REVIEW

File: src/annotate.rs [hunk 1]
```diff
@@ -27,6 +27,7 @@ fn prepare_metadata(wasm_path: PathBuf, metadata_path: PathBuf) -> Result<Metada
         .map_err(|e| anyhow!("Cannot compute ProtocolVersion used by the policy: {:?}", e))?;
 
     metadata.protocol_version = Some(protocol_version);
+    metadata.annotated_by = Some(format!("kwctl v{}", env!("CARGO_PKG_VERSION")));
```
Comment (ereslibre) [inline src/annotate.rs:31]: If we need to process this on a later operation I think it better to write `CARGO_PKG_VERSION`, so we can just use a semver parser directly and do comparisons.

**Review by ereslibre — CHANGES_REQUESTED** [review]

File: src/annotate.rs (hunk 1 above)
Comment (flavio) [reply src/annotate.rs:47]: resolved as discussed on slack (see below)

**Comment by flavio** [comment]
I've done the following change to the PR:  kwctl creates an annotation inside of the policy's annotations to store its version.

The annotation is named `io.kubewarde.kwctl` and the version of kwctl is stored as value.

File: src/inspect.rs [hunk 2]
```diff
@@ -78,12 +78,12 @@ impl MetadataPrettyPrinter {
...
-            "io.kubewarden.policy.author",
-            "io.kubewarden.policy.url",
-            "io.kubewarden.policy.source",
-            "io.kubewarden.policy.license",
+            crate::constants::ANNOTATION_POLICY_TITLE,
+            crate::constants::ANNOTATION_POLICY_DESCRIPTION,
+            crate::constants::ANNOTATION_POLICY_AUTHOR,
+            crate::constants::ANNOTATION_POLICY_URL,
+            crate::constants::ANNOTATION_POLICY_SOURCE,
+            crate::constants::ANNOTATION_POLICY_LICENSE,
```
Comment (ereslibre) [inline src/inspect.rs:86]: By `use crate::constants` we could remove some of the duplication here and use `constants::THE_CONST`

**Review by ereslibre — APPROVED** [review]
LGTM, non-blocking bike shedding

File: src/inspect.rs (hunk 2 above)
Comment (flavio) [reply src/inspect.rs]: fixed

**Pull request merged**